packages/jazz-tools/src/tools/implementation/zodSchema/schemaTypes/GroupSchema.ts

```diff
@@ -0,0 +1,73 @@
+import { Group } from "../../../coValues/group.js";
+import {
+  ID,
+  SubscribeListenerOptions,
+  SubscribeRestArgs,
+} from "../../../coValues/interfaces.js";
+import {
+  Account,
+  RefsToResolve,
+  RefsToResolveStrict,
+  Resolved,
+  ResolveQuery,
+} from "../../../internal.js";
+import { CoreCoValueSchema } from "./CoValueSchema.js";
+import { coOptionalDefiner } from "../zodCo.js";
+import { CoOptionalSchema } from "./CoOptionalSchema.js";
+
+export interface CoreGroupSchema extends CoreCoValueSchema {
+  builtin: "Group";
+}
+
+export function createCoreGroupSchema(): CoreGroupSchema {
+  return {
+    collaborative: true as const,
+    builtin: "Group" as const,
+  };
+}
+
+export class GroupSchema implements CoreGroupSchema {
+  readonly collaborative = true as const;
+  readonly builtin = "Group" as const;
+
+  constructor(private coValueClass: typeof Group) {}
+
+  getCoValueClass(): typeof Group {
+    return this.coValueClass;
+  }
```

Comment:
I'd just make this simpler and return Group
```suggestion
  getCoValueClass(): typeof Group {
    return Group;
  }
```

Reply:
84b08c3fd